netbox/utilities/templatetags/helpers.py

```diff
@@ -230,14 +230,22 @@ def querystring(request, **kwargs):
 
 
 @register.inclusion_tag('utilities/templatetags/utilization_graph.html')
-def utilization_graph(utilization, warning_threshold=75, danger_threshold=90):
+def utilization_graph(data_input, warning_threshold=75, danger_threshold=90):
```

Comment:
`data_input` should be split into two discrete arguments (utilization and total). Note that this will involve updating all usages of the tag within the project.